### PR TITLE
Add Xcode project [to `add-framework-support` branch]

### PR DESCRIPTION
**Pull request by rastersize**
This is equivalent to, and supersedes‚ #151 but instead targets the `add-framework-support` branch.

### Todo
- [x] Integrate framework with the **ACME Browser** demo app.

### CODE REVIEW

File: Info.plist [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>CFBundleDevelopmentRegion</key>
+	<string>en</string>
+	<key>CFBundleExecutable</key>
+	<string>$(EXECUTABLE_NAME)</string>
+	<key>CFBundleIdentifier</key>
+	<string>com.agilebits.$(PRODUCT_NAME:rfc1034identifier)</string>
```
Comment (radazzouz) [inline Info.plist]: We should change the bundle identifier for this branch. This way we can run and test the demo apps side by side. We appended `low-level` after `com.agilebits.` in the Low Level API branch. We should append `framework` or something like this for this branch.

The bundle identifier should be `com.agilebits.framework.$(PRODUCT_NAME:rfc1034identifier)` for both **ACME** and **ACME Browser**

**Comment by radazzouz** [comment]
I looks great. Thanks @rastersize :+1: 

We should change the bundle identifiers or both **ACME** and **ACME Browser** in this branch. This way we can run and test the demo apps side by side. We appended `low-level` to `com.agilebits.` in the Low Level API branch. We should append `framework` or something like this for this branch.

The bundle identifiers should be `com.agilebits.framework.$(PRODUCT_NAME:rfc1034identifier)`

**Comment by rastersize** [comment]
I’ve changed the bundle ids of the demo apps :smiley_cat:

**Comment by radazzouz** [comment]
Thanks so much for the awesome work @rastersize!

I just tested your PR with both the stable and the beta versions of 1Password 5 for iOS and it all seems to work fine.

I'll be merging this PR and PR #153.

Cheers!

**Pull request merged**